edk2toollib/windows/capsule/inf_generator.py

```diff
@@ -331,6 +332,7 @@ def MakeInf(self, OutputInfFilePath: os.PathLike, FirmwareBinFileName: str, Roll
             EsrtGuid=self.EsrtGuid,
             Provider=self.Provider,
             MfgName=self.Manufacturer,
+            OsBuildVersion=OS_BUILD_VERSION_DIRID13_SUPPORT,
```

Comment:
This variable should similarly be called `OsBuildVersionDirId13Support` or something that allows it to carry that value forward 1:1 so it tracks the OS build version for that specific need.

Reply:
we could be restricting the version for other reasons too in the future, with another update, so I left that flexible, since I had an opportunity to do so in this file, while also indicating using the define name as to why is the version restriction in place.